src/app/js/formats/bubbleChart/BubbleAdmin.js

```diff
@@ -50,20 +51,32 @@ class BubbleAdmin extends Component {
         args: defaultArgs,
     };
 
+    constructor(props) {
+        super(props);
+        this.handleColorsChange = this.handleColorsChange.bind(this);
```

Comment:
I don't get the reason why you need to bind `this.handleColorsChange`, but not `this.setDiameter`, nor `this.setParams`.

Reply:
> Binding methods helps ensure that the second snippet works the same way as the first one.
([React docs](https://reactjs.org/docs/faq-functions.html#why-is-binding-necessary-at-all))

It is necessary to link the parent and the new child Component `ColorPickerParamsAdmin` !

Reply:
It's used in `onColorsChange`, and not in `onChange`, like in the other cases.

OK